web_src/js/features/repo-issue-list.js

```diff
@@ -203,3 +203,13 @@ export function initRepoIssueList() {
   initRepoIssueListAuthorDropdown();
   initIssuePinSort();
 }
+
+export function initHighlightArchivedLabels() {
+  if (!document.querySelectorAll('.show-archived-labels-checkbox').length) return;
+  document.querySelector('.show-archived-labels-checkbox')
+    .addEventListener('input', () => {
+      for (const archivedLabel of document.querySelectorAll('.archived-label')) {
+        archivedLabel.classList.toggle('gt-hidden');
```

Comment:
```suggestion
        toggleElem(archivedLabel, e.target.checked);
```

Reply:
`toggleElem(archivedLabel, e.target.checked);` I will call it Black Magic